src/main/scala/DbHelper.scala

```diff
@@ -39,7 +39,9 @@ object DbHelper {
     val sql =
```

Comment:
`old_id`  field in `text` table is actually foreign field to `revision` table, not the `page` table.